command_test.go

```diff
@@ -1245,15 +1245,15 @@ func TestCommand_Float64Flag(t *testing.T) {
 }
 
 func TestCommand_ParseSliceFlags(t *testing.T) {
-	var parsedIntSlice []int64
+	var parsedIntSlice []int
 	var parsedStringSlice []string
 
 	cmd := &Command{
 		Commands: []*Command{
 			{
 				Name: "cmd",
 				Flags: []Flag{
-					&IntSliceFlag{Name: "p", Value: []int64{}, Usage: "set one or more ip addr"},
```

Comment:
Nitpick you could have changed this to Int64SliceFlag and avoided all the 
int64->int changes. Same for other places.